sharding/client/client.go

```diff
@@ -93,42 +100,22 @@ func (c *collatorClient) Start() error {
 		return fmt.Errorf("cannot unlock account. %v", err)
 	}
 
-	if err := initSMC(c); err != nil {
-		return err
-	}
-
-	// Deposit 100ETH into the collator set in the SMC. Checks if account
-	// is already a collator in the SMC (in the case the client restarted).
-	// Once that's done we can subscribe to block headers.
-	//
-	// TODO: this function should store the collator's SMC index as a property
-	// in the client's struct
-	if err := joinCollatorPool(c); err != nil {
+	smc, err := initSMC(c)
```

Comment:
I don't know if executor node uses SMC, we might have to change this for phase 3

Reply:
The executor node won't use this client anyway, it doesn't need main chain communication, right?

Reply:
That's true